### DISCUSSION TITLE
Are there ways to exchange/clone repositories without git?

### DISCUSSION THREAD

**Discussion by mmlanger** [jj-vcs/jj#3939] [Q&A] [answered]
Ok, I think I’ve got a good grasp of the basics and am really impressed with how nicely Jujutsu works.

What I am wondering now is how to properly exchange repositories without using Git. As far as I understand, I can surely just copy the repo to a location, and someone else could also just paste their changes. The working copy might not be what it should be, but all the changes are there as Jujutsu’s model seems to allow this.

What I miss is an easier method to exchange/clone/push/pull repositories besides a Git server. Is there something like a ‘fossil server’ or ‘hg serve’ in Jujutsu? Rsync works, but it is a bit of a hassle, especially if I have to exchange e.g. between a Mac and a Windows machine, as rsync is usually not easily available on Windows.

A passage in the documentation on how workflows in pure Jujutsu could look would be nice.

**Comment by PhilipMetzger** [comment, marked as answer]
> What I miss is an easier method to exchange/clone/push/pull repositories besides a Git server.  Is there something like a ‘fossil server’ or ‘hg serve’ in Jujutsu?

No, as it depends on the native repository, which currently just is a proof-of-concept instead of a real backend™. And some kind of server for it.  

> A passage in the documentation on how workflows in pure Jujutsu could look would be nice.

It is hard to write a passage about something which currently doesn't exist.

**Reply by mmlanger** [reply]
I see, too early. Thanks for the reply!
